### ISSUE TITLE
Understand gradient clipping

### ISSUE THREAD

**Issue by ldoshi** [ldoshi/rome-wasnt-built-in-a-day#42]
Switching from loss clipping to gradient clipping made a big difference. 

Understand better if we need both or one is sufficient. 

Is the value of in config/training.py sufficient? 

Should other alternatives be considered?

https://pytorch-lightning.readthedocs.io/en/stable/advanced/training_tricks.html
